spec/shared_examples/pcp_resources.rb

```diff
@@ -1,5 +1,7 @@
 shared_examples_for 'pcp::resources' do |_facts|
   it { is_expected.to have_pcp__pmda_resource_count(0) }
+  it { is_expected.to have_pcp__pmie_resource_count(0) }
+  it { is_expected.to have_pcp__pmlogger_resource_count(0) }
```

Comment:
There is one pmie and one pmlogger resource applied by default

```suggestion
  it { is_expected.to have_pcp__pmie_resource_count(1) }
  it { is_expected.to have_pcp__pmlogger_resource_count(1) }
```